tests/testthat/test-DiseasystoreGoogleCovid19.R

```diff
@@ -1,6 +1,19 @@
 test_that("DiseasystoreGoogleCovid19 works", {
 
-  # First we create a temporary directory for the Google COVID-19 data
+  # Check we can use the Google API directly
```

Comment:
`ds` is not used for the remainder of this test (>150 lines of code).

[Tests should be specific (😇)](https://github.com/ssi-dk/diseasy/wiki/Diseasy-coding-standard#5-tests-should-be-specific). I strongly suggest that you write a separate test for this.

Reply:
Created a separate issue https://github.com/ssi-dk/diseasystore/issues/63 for this to keep track of this.